elibrary-hibernate/src/main/resources/db/changelog/truncate-all-cascade-changelog.xml

```diff
@@ -5,7 +5,7 @@
 	<changeSet author="Jakub.Rutkowski" id="0000000019">
 		<comment>Clear any existing legacy data in the tables</comment>
 		<sql splitStatements="true">
-			TRUNCATE TABLE public.com_authorization CONTINUE IDENTITY CASCADE;
+			-- TRUNCATE TABLE public.com_authorization CONTINUE IDENTITY CASCADE;
```

Comment:
Please revert as it's the change only for the local development env. 🗑️

Reply:
Fixed 🔨